pandas/tests/frame/test_sorting.py

```diff
@@ -321,7 +342,11 @@ def test_sort_nat_values_in_int_column(self):
         assert_frame_equal(df_sorted, df_reversed)
 
         df_sorted = df.sort_values(["datetime", "float"], na_position="last")
-        assert_frame_equal(df_sorted, df_reversed)
+        assert_frame_equal(df_sorted, df)
```

Comment:
As far as I understand, this was a bug in the test.  Previously, the two assertions implied that na_position does not have any effect on the sort, which is incorrect.